app/serializers/assignment_repo_serializer.rb

```diff
@@ -13,5 +14,10 @@ def username
   def displayName
     object.user.github_user.name || ""
   end
+
+  def rosterIdentifier
+    return nil unless instance_options[:roster]
+    instance_options[:roster].roster_entries.find_by(user_id: object.user.id)&.identifier
```

Comment:
Is this an N+1?

Reply:
The results of the `roster.roster_entries` query would _probably_ be cached, but I changed this to pass in roster entries instead of roster so we don't need to make this query every time.